### PR TITLE
Add AddDatasourceCommand

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/inventory/InventoryIdUtil.java [hunk 1]
```diff
@@ -25,6 +25,16 @@
...
+
+        public String getFeedId() {
+            return feedId;
+        }
+        public String getManagedServerName() {
+            return managedServerName;
+        }
+        public String getIdPart() {
+            return idPart;
+        }
```
Comment (jmazzitelli) [inline hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/inventory/InventoryIdUtil.java:37]: if you want to make public getters, then make the data fields private so it forces people to go through the getters.

Comment (ppalaga) [inline hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/inventory/InventoryIdUtil.java:37]: Yeah, wanted to do that in a separate commit, but... forgot. Will do.
